### PR TITLE
feat: Display "No events liked"

### LINKED ISSUE #630: Displaying a message when there are no liked events.
**Describe the chore**
When there are no liked events then the fragment is blank, at least it should display "There are no liked events" .
Same thing can be done with Tickets fragment too.

**Would you like to work on the issue?**

- [X] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by Abhimanyu121**
Fixes #630 

Changes: Now it displays "No liked Events" when there are no liked events.

Screenshots for the change:
better

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -136,5 +136,7 @@
     <!--settings profile dialog-->
     <string name="message">Are you sure you want to log out?</string>
     <string name="cancel">Cancel</string>
+    <string name="no_liked_events">No Liked Events</string>
+    <string name="no_tickets">No Tickets</string>
```
Comment (nikit19) [inline app/src/main/res/values/strings.xml]: This string is not used anywhere right ?

File: app/src/main/res/layout/fragment_favorite.xml [hunk 2]
```diff
@@ -6,6 +6,19 @@
     <FrameLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent">
+       <LinearLayout
+           android:layout_width="match_parent"
+           android:layout_height="wrap_content"
+           android:orientation="vertical">
+           <TextView
```
Comment (nikit19) [inline app/src/main/res/layout/fragment_favorite.xml]: The text would look better IMO if it is both horizontally and vertically centred.

Comment (Abhimanyu121) [reply app/src/main/res/layout/fragment_favorite.xml]: Okay, I will resolve this

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (Abhimanyu121) [reply app/src/main/res/values/strings.xml]: "no_tickets" string was supposed to be used to show "no tickets " in tickets fragment . But I couldn't fix it properly, I will try it after some time .

File: app/src/main/res/layout/fragment_favorite.xml (hunk 2 above)
Comment (Abhimanyu121) [reply app/src/main/res/layout/fragment_favorite.xml]: screenshot updated

**Comment by simarsingh24** [comment]
screenshot_20181014-141811_eventbrite
please follow this

**Comment by Abhimanyu121** [comment]
> 
> 
> screenshot_20181014-141811_eventbrite
> please follow this

okay , gonna make something similar :D

**Comment by Abhimanyu121** [comment]
screenshot updated

File: app/src/main/java/org/fossasia/openevent/general/favorite/FavoriteFragment.kt [hunk 3]
```diff
@@ -60,6 +61,9 @@ class FavoriteFragment : Fragment() {
             it?.let {
                 favoriteEventsRecyclerAdapter.addAll(it)
                 favoriteEventsRecyclerAdapter.notifyDataSetChanged()
+                if(favoriteEventsRecyclerAdapter.itemCount!=0)
```
Comment (simarsingh24) [inline app/src/main/java/org/fossasia/openevent/general/favorite/FavoriteFragment.kt]: space after if

File: app/src/main/java/org/fossasia/openevent/general/favorite/FavoriteFragment.kt [hunk 4]
```diff
@@ -60,6 +61,9 @@ class FavoriteFragment : Fragment() {
...
+                    noLikedText.visibility=View.GONE
```
Comment (simarsingh24) [inline app/src/main/java/org/fossasia/openevent/general/favorite/FavoriteFragment.kt]: space before and after `=`

**Comment by Abhimanyu121** [comment]
@iamareebjamal are you referring to me writing if statement in that "let" block ? If not, can you please tell me what exactly are you referring to ?

File: app/src/main/java/org/fossasia/openevent/general/favorite/FavoriteFragment.kt [hunk 5]
```diff
@@ -60,6 +62,9 @@ class FavoriteFragment : Fragment() {
             it?.let {
                 favoriteEventsRecyclerAdapter.addAll(it)
                 favoriteEventsRecyclerAdapter.notifyDataSetChanged()
+                if (favoriteEventsRecyclerAdapter.itemCount!=0)
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/favorite/FavoriteFragment.kt]: This

File: app/src/main/java/org/fossasia/openevent/general/favorite/FavoriteFragment.kt [hunk 6]
```diff
@@ -43,6 +44,7 @@ class FavoriteFragment : Fragment() {
                 bundle.putLong(EVENT_ID, eventID)
                 fragment.arguments = bundle
                 activity?.supportFragmentManager?.beginTransaction()?.add(R.id.rootLayout, fragment)?.addToBackStack(null)?.commit()
+
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/favorite/FavoriteFragment.kt]: Extra

File: app/src/main/java/org/fossasia/openevent/general/favorite/FavoriteFragment.kt [hunk 7]
```diff
@@ -51,7 +53,7 @@ class FavoriteFragment : Fragment() {
                 favoriteEventViewModel.setFavorite(event.id, !isFavourite)
                 event.favorite = !event.favorite
                 favoriteEventsRecyclerAdapter.notifyItemChanged(id)
-            }
+                }
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/favorite/FavoriteFragment.kt]: Extra

**Comment by Abhimanyu121** [comment]
please review

**Comment by Abhimanyu121** [comment]
i squashed all the commits for convenience and also tried to fix styling , but if it still isn't proper can you please point out what mistakes i did ?

**Pull request merged**